site/content/en/spec-3.2.md

```diff
@@ -76,6 +60,28 @@ These are the definitions for some of the terms used in this specification:
 - TOML --- A configuration file format as available at
   <https://toml.io/en/v1.0.0>.
 
+## Covered Files
```

Comment:
Thinking loudly: The section mostly contains files that are not covered, so perhaps renaming the section might make it easier for quick readers who ignore the first paragraph?

Reply:
Renamed the section to 'Covered and ignored Files'. Thanks @mxmehl !